packages/web-components/src/components/va-alert/va-alert.scss

```diff
@@ -66,3 +67,18 @@
     left: 12px;
   }
 }
+
+.usa-site-alert .usa-alert.usa-alert--warning {
+  background-color: var(--vads-color-warning-lighter);
+  border-left-color: var(--vads-color-warning);
+}
+
+.usa-site-alert .usa-alert.usa-alert--success {
+  background-color: var(--vads-color-success-lighter);
+  border-left-color: var(--vads-color-success);
+}
+
+.usa-site-alert .usa-alert.usa-alert--error {
+  background-color: var(--vads-color-error-lighter);
+  border-left-color: var(--vads-color-error);
+}
```

Comment:
The USWDS `usa-site-alert` package only supports the **info** color variation. This is probably intentional because the [site alert docs](https://designsystem.digital.gov/components/site-alert/) don't show them being options.

I'm assuming we want more flexibility to be able to use site wide alerts for warning, success, and error too. For that reason, I needed to add those colors here.